backend/src/Entity/ChecklistItem.php

```diff
@@ -2,14 +2,20 @@
 
 namespace App\Entity;
 
-use App\Repository\ChecklistRepository;
+use App\Repository\ChecklistItemRepository;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
- * @ORM\Entity(repositoryClass=ChecklistRepository::class)
+ * @ORM\Entity(repositoryClass=ChecklistItemRepository::class)
  */
-class Checklist
+class ChecklistItem
 {
+
+    const TYPE_DAY = "day";
+    const TYPE_WEEK = "week";
+    const TYPE_MONTH = "month";
+    const TYPE_HUNDREDDAYS = "100days";
```

Comment:
```suggestion
    const TYPE_HUNDRED_DAYS = "100days";
```
or (I think this is more specific)

```suggestion
    const TYPE_ONE_HUNDRED_DAYS = "100days";
```